### PR TITLE
Extend the DBus documentation of the Storage module

### PR DESCRIPTION
**Pull request by poncovka**
Provide more information about the available methods and how they are connected.

### CODE REVIEW

File: pyanaconda/modules/storage/storage_interface.py [hunk 1]
```diff
@@ -81,7 +91,16 @@ def CreatedPartitioning(self) -> List[ObjPath]:
...
+        Choose a valid partitioning layout of the specified partitioning
+        module for an installation. Call InstallWithTasks to execute the
+        scheduled actions and commit these changes to selected disks.
+
+        The device tree module provides information about the partitioned
+        storage model instead of the model of the current storage if there
+        is an applied partitioning.
+
         :param partitioning: a path to a partitioning
+        :raise: InvalidStorageError of the partitioning is not valid
```
Comment (VladimirSlavik) [inline pyanaconda/modules/storage/storage_interface.py:103]: s/of/if/ ?

**Review by VladimirSlavik — APPROVED** [review]
Thank you! Very useful.

File: pyanaconda/modules/storage/storage_interface.py (hunk 1 above)
Comment (poncovka) [reply pyanaconda/modules/storage/storage_interface.py]: Fixed.

**Review by KKoukiou — APPROVED** [review]
Thank you!

**Pull request merged**